cmd/docker/docker.go

```diff
@@ -175,6 +176,16 @@ func main() {
 	}
 }
 
+func isContentTrustEnabled() bool {
```

Comment:
nit: the `is` prefix is a bit redundant because it already has an `Enabled` suffix; it reads equally well without (`if contentTrustEnabled()`)